src/qt/overviewpage.cpp

```diff
@@ -208,7 +208,7 @@ void OverviewPage::UpdateBoincUtilization()
     ui->labelProject->setText(QString::fromUtf8(GlobalStatusStruct.project.c_str()));
     ui->labelCpid->setText(QString::fromUtf8(GlobalStatusStruct.cpid.c_str()));
     ui->labelStatus->setText(QString::fromUtf8(GlobalStatusStruct.status.c_str()));
-    ui->labelPoll->setText(QString::fromUtf8(GlobalStatusStruct.poll.c_str()));
+    ui->labelPoll->setText(QString::fromUtf8(GlobalStatusStruct.poll.c_str()).replace(QChar('_'),QChar(' '), Qt::CaseSensitive));
```

Comment:
Do you have to explicitly cast to QChar or will its constructor do?
`ui->labelPoll->setText(QString::fromUtf8(GlobalStatusStruct.poll.c_str()).replace('_', ' ', Qt::CaseSensitive));`

Reply:
Yes, it wouldn't compile like that.

Reply:
Weird, but I take your word for it :)